templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
```

Comment:
Assignments don't *need* quoting, but I often quote them anyway because it doesn't hurt. Up to you what you want to do here, the minimum, safe, POSIX, thing would be:

```sh
commits=$("$git" rev-list HEAD | wc -l)
```